app/components/theme/lib/theme-map.css

```diff
@@ -167,20 +168,20 @@
  */
 
 :root {
-  --Dropdown-content-color-bg         : var(--color-white);
-  --Dropdown-content-border           : var(--default-border);
-  --Dropdown-content-z                : var(--z-scale-7);
-  --Dropdown-title-size               : calc(14/16 * 1rem);
-  --Dropdown-title-color              : color(var(--color-secondary) tint(40%));
-  --Dropdown-title-color-bg           : transparent;
-  --Dropdown-item-color               : var(--color-highlight);
-  --Dropdown-item-hover-color         : var(--Dropdown-content-color-bg);
-  --Dropdown-item-hover-color-bg      : var(--Dropdown-item-color);
-  --Dropdown-item-transition          : all .2s ease-out;
-  --Dropdown-rhythm                   : var(--rhythm);
-  --Dropdown-transition               : all .2s var(--transition-ease-out-back);
-  --Dropdown-z                        : var(--z-scale-1);
-  --Dropdown-z-active                 : var(--z-scale-8);
+  --Dropdown-rhythm              : var(--rhythm);
+  --Dropdown-transition          : all .2s var(--transition-ease-out-back);
+  --Dropdown-z-index             : var(--z-scale-1);
+  --Dropdown-active-z-index      : var(--z-scale-9);
+  --Dropdown-content-color-bg    : var(--color-white);
+  --Dropdown-content-border      : var(--default-border);
+  --Dropdown-content-z-index     : var(--z-scale-8);
+  --Dropdown-title-size          : calc(14/16 * 1rem);
+  --Dropdown-title-color         : color(var(--color-secondary) tint(40%));
+  --Dropdown-title-color-bg      : transparent;
+  --Dropdown-item-color          : var(--color-highlight);
+  --Dropdown-item-hover-color    : var(--Dropdown-content-color-bg);
+  --Dropdown-item-hover-color-bg : var(--Dropdown-item-color);
+  --Dropdown-item-transition     : all .2s ease-out;
```

Comment:
Is this being auto-formatted and rearranged? This could make for a lot of extra work in reviews if that is happening and we don't prevent it.